tests/test_create_sentence.rs

```diff
@@ -27,3 +35,41 @@ fn test_post_sentence_returns_200() {
         StatusCode::Created,
     );
 }
+
+#[test]
+fn test_post_sentence_with_used_uuid_returns_409() {
+
+    let connection = db::get_connection();
+    db::clear(&connection);
+
+    let sentence_uuid = uuid::Uuid::new_v4();
+    let sentence_text = "This is one sentence";
+    let sentence_iso639_3 = "eng";
+    db::insert_sentence(
+        &connection,
+        &sentence_uuid,
+        &sentence_text,
+        &sentence_iso639_3,
+    );
+
+    let mut json = HashMap::new();
+    json.insert("id", sentence_uuid.to_string());
+    json.insert("text", sentence_text.to_string());
+    json.insert("iso639_3", sentence_iso639_3.to_string());
```

Comment:
put a different text and language to be sure it comes from the uuid and not the fact that the couple lang/text is the same (as it's also supposed to do 409 )